tox.ini

```diff
@@ -12,4 +13,13 @@ deps =
 
 commands =
     pylama mozdownload
-    py.test {posargs}
+    coverage run --source mozdownload -m py.test {posargs}
+
+[testenv:clean]
+commands=
+  coverage erase
+
+[testenv:stats]
+commands=
+    coverage report --omit='.tox/*'
+    coverage html --omit='.tox/*'
```

Comment:
Yes, it will give more details. Essentially the index.html in the htmlcov directory shows the same as the CLI output, but each file is a link to the code in the file indicating which code was covered and which one was not.